mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -218,6 +218,7 @@ def handle_roi_clicked(self, roi: SpectrumROI) -> None:
             self.view.table_view.current_roi_name = roi.name
             self.view.table_view.last_clicked_roi = roi.name
             self.view.set_roi_properties()
+            self.view.set_selected_roi(roi.name)
```

Comment:
If I click on an ROI on the image view I get
```
ERROR: Traceback (most recent call last):
  File "/home/sam/git/mantidimaging/mantidimaging/gui/windows/spectrum_viewer/presenter.py", line 221, in handle_roi_clicked
    self.view.set_selected_roi(roi.name)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'SpectrumViewerWindowView' object has no attribute 'set_selected_roi'. Did you mean: 'selected_row'?
```
This line can go, because we don't need to change the ROI in `ROISelectionWidget` when things happen on the first tab.

That means changes in `presenter_test.py` can also be removed.